Test/git-issues/git-issue-1231.dfy

```diff
@@ -0,0 +1,11 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+datatype Dt = Dt(x:int)
```

Comment:
While you're at it, please also add a similar test for `codatatype` and `newtype`.

Reply:
Done.